service/src/test/resources/ehr/mapper/procedurerequest/procedure-request-resource-0.json

```diff
@@ -0,0 +1,40 @@
+{
+    "resourceType": "ProcedureRequest",
+    "id": "1B1C6F50-D8BE-4480-83D6-EF25AC5F1836",
+    "status": "active",
+    "intent": "proposal",
+    "occurrenceDateTime": "2010-07-14T16:32:51.42+01:00",
+    "authoredOn": "2010-01-13T15:29:50.1+00:00",
+    "requester": {
+        "agent": {
+            "reference": "Organization/6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73",
+            "display": "Organisation name"
```

Comment:
Were theses fixes derived from actual examples within our test data set? I ask because I cannot find any usages of agent.display in our datasets.

Reply:
the excel with mapping says that there won't be all examples of the data and we will need to prepare them on our own, but that display field was wrong assumption. The mapping was changed to search for referenced resources in the all input bundle, and the information there will be used.